### PR TITLE
feat(testing): add test for `parse` when error in args + config

### LINKED ISSUE #4858: [Testing]: add more coverage for src/node/cli.ts
For #TestingMondays, I'd like to increase our coverage for `src/node/cli.ts`. 293 is missing coverage.

We already have tests in place for `parse`.

To test this, we should create a test which uses a config file and adds an option that doesn't exist. We expect the function to throw an error with a specific message.

### PR DESCRIPTION
**Pull request by jsjoeio**
This adds a test for `parse` to hit a line where an error is thrown with a bad arg and a config file path are passed in.

Fixes #4858

### CODE REVIEW

File: test/unit/node/update.test.ts [hunk 1]
```diff
@@ -218,8 +218,8 @@ describe("update", () => {
   it("should reject if response has status code 500", async () => {
     if (isAddressInfo(_address)) {
       const mockURL = `http://${_address.address}:${_address.port}/reject-status-code`
-      let provider = new UpdateProvider(mockURL, settings())
-      let update = await provider.getUpdate(true)
+      const provider = new UpdateProvider(mockURL, settings())
```
Comment (jsjoeio) [inline test/unit/node/update.test.ts:221]: Very strange...I didn't make this change but when I ran `yarn lint` or `yarn fmt` locally, this changed happened?

Comment (code-asher) [reply test/unit/node/update.test.ts:221]: Huh I think we are passing `--fix` to auto-fix but I did not know they would change `let` to `const`.

Comment (jsjoeio) [reply test/unit/node/update.test.ts:221]: I didn't either! Maybe a new thing? Or maybe I did something janky...no idea

Comment (code-asher) [reply test/unit/node/update.test.ts:221]: Could be something new!  Either way, const looks right since we are not changing these values.

**Review by code-asher — APPROVED** [review]

**Pull request merged**